### ISSUE TITLE
Does this framework support full parameter PPO tuning for the Qwen2.5-14B model on 8-A100 GPUs with 80GB memory each?

### ISSUE THREAD

**Issue by hljjjmssyh** [volcengine/verl#40]
as mentioned in the title.

**Comment by PeterSH6** [comment]
Hi @hljjjmssyh, I think our framework can support your needs. You can try using FSDP backend + vLLMRollout with `tensor_parallel_size=8` and tune the `gpu_memory_utilization` and other hyper-parameters.

If you encounter OOM, you can turn on the param offload in reference and reward model similar to [qwen2_7b example](https://github.com/volcengine/verl/blob/main/examples/ppo_trainer/run_qwen2-7b.sh). And you can also try param/grad/optimzier offload in actor/critic model if you prefer a larger micro-batch size for training.

**Comment by PeterSH6** [comment]
Now, you can refer to the performance tuning guide to tune the RL training: https://verl.readthedocs.io/en/latest/perf/perf_tuning.html

**Comment by sin2y** [comment]
> Hi @hljjjmssyh, I think our framework can support your needs. You can try using FSDP backend + vLLMRollout with `tensor_parallel_size=8` and tune the `gpu_memory_utilization` and other hyper-parameters.
> 
> If you encounter OOM, you can turn on the param offload in reference and reward model similar to [qwen2_7b example](https://github.com/volcengine/verl/blob/main/examples/ppo_trainer/run_qwen2-7b.sh). And you can also try param/grad/optimzier offload in actor/critic model if you prefer a larger micro-batch size for training.

Hi, is tensor_parallel effective for the actor during the policy update? Or only during rollout?
